test/ui-test/testSuites/suite_status/shared/scripts/names.py

```diff
@@ -151,3 +150,19 @@
 walletSettingsLineButton = {"container": mainWindow_ScrollView_2, "objectName": "WalletSettingsLineButton", "type": "StatusSettingsLineButton", "visible": True}
 navBarListView_Settings_navbar_StatusNavBarTabButton = {"checkable": True, "container": mainWindow_navBarListView_ListView, "objectName": "Settings-navbar", "type": "StatusNavBarTabButton", "visible": True}
 settings_navbar_settings_icon_StatusIcon = {"container": navBarListView_Settings_navbar_StatusNavBarTabButton, "objectName": "settings-icon", "source": "qrc:/StatusQ/src/assets/img/icons/settings.svg", "type": "StatusIcon", "visible": True}
+
+# Main Window - chat related:
+mainWindow_chatItem = {"container": mainWindow_ScrollView, "objectName": "chatItem", "type": "StatusChatListItem", "visible": True}
```

Comment:
nit: for this selector and all below, you don't need the visible attribute

Reply:
It's possible that they are needed if there are multiple channels in the list, since we only want the active and visible chat

Reply:
Tested. Not needed, so removed!